src/test_typing_extensions.py

```diff
@@ -4494,6 +4494,11 @@ def test_pickle(self):
                 unpickled = pickle.loads(pickled)
                 self.assertIs(unpickled, Alias)
 
+    def test_no_instance_subclassing(self):
+        with self.assertRaises(TypeError):
+            class MyAlias(TypeAliasType):
+                pass
```

Comment:
(Might be worth adding a test like this to https://github.com/python/cpython/pull/103764 as well; I couldn't spot one)

Reply:
Good point, https://github.com/python/cpython/pull/103764/commits/d4e72a5669a6a11b17c5591fe1494f29e25f963d